src/esm_tests/cli.py

```diff
@@ -142,12 +151,17 @@ def main():
         sys.exit(1)
 
     # Get user info for testing
-    info = user_config(info)
+    user_config(info)
 
     # User-specific Info to remove from the files ``last_tested`` files
+    info["mnt"] = ""
+    pwd = os.getcwd()
+    if pwd.startswith("/mnt/lustre"):
+        info["mnt"] = "/".join(pwd.split("/")[:3])
     info["rm_user_info"] = {
-        "ACCOUNT": info["user"]["account"],
         "TEST_DIR": info["user"]["test_dir"],
+        "HOME_DIR": f"{os.path.expanduser('~')}",
+        #"ACCOUNT": info["user"]["account"],
```

Comment:
Could be included as `.get("account")` and then this defaults to `None`

Reply:
`ACCOUNT` should be removed here as it's not in used and shouldn't be used anymore.